step-controller/step-controller-remote-client/src/main/java/step/client/collections/remote/RemoteCollection.java

```diff
@@ -176,7 +176,13 @@ protected UnsupportedOperationException notImplemented()  {
         return new UnsupportedOperationException("This method is currently not implemented");
     }
 
-	@Override
+    @Override
+    // Empty implementation to be replaced
```

Comment:
I don't understand this change or the comment :-p

Reply:
Hehe, I had indeed problem to explain it in a few words